### ISSUE TITLE
Deploy eak-map-styles

### ISSUE THREAD

**Issue by maxammann** [ehrenamtskarte/ehrenamtskarte#244]
@elKei24 created the eak-map-styles package. This needs to be deployed. Also nginx and salt config needs to be adjusted

**Comment by elKei24** [comment]
@maxammann What do you mean with "deployed"? The package should be pushed to the server automatically once the PR is merged.

**Comment by maxammann** [comment]
Yeah the configuration afterwards, like installing configuring etc.

**Comment by maxammann** [comment]
this is already done
